documentation/memory/step3/using_gc_traces.md

```diff
@@ -20,4 +20,18 @@ v8.setFlagsFromString('--trace_gc');
 setTimeout(() => { v8.setFlagsFromString('--notrace_gc'); }, 60e3);
 ```
 
-## Examining a trace
\ No newline at end of file
+## Examining a trace
+
+Obtained traces of garbage collection looks like the following lines.
+
+```
+[19278:0x5408db0]  44 ms: Scavenge 2.3 (3.0) -> 1.9 (4.0) MB, 1.2 / 0.0 ms  (average mu = 1.000, current mu = 1.000) allocation failure
+
+[23521:0x10268b000]  120 ms: Mark-sweep 100.7 (122.7) -> 100.6 (122.7) MB, 0.15 / 0.0 ms  (average mu = 0.132, current mu = 0.137) deserialize GC in old space requested
+```
+
+This is how to interpret the trace data:
+
+```
+[PID: isolate] < time taken since GC started in ms> : < type/phase of GC > <heap used before GC call in MB> ( < allocated heap before GC call in MB > ) -> < heap used after GC in MB> ( < allocated heap after GC in MB>) <time spent in GC in ms> [ < reason for GC >]
```

Comment:
This is hard to read. A graphic where we point to the sections and say what they are might work better.

If we have examples of how to use the traces to identify problems that would also be good.  For example in terms of running out of memory.  If you first see that that the old space size is continually increasing but it takes too long to hit the max, setting the max old space size to a smaller value can help. 

Another one is that if you see that the time spent in GC is continually increasing or is a large portion of the overall that can mean you are short on memory even if you don't OOM.

Reply:
@mhdawson, Modified according to your suggestion. PTAL.